backend/scripts/youtube/youtube_channel_oauth_flow.py

```diff
@@ -0,0 +1,59 @@
+"""
+Script to generate credentials to be authenticated on Youtube API
+as the owner of a Youtube channel.
+
+Follow these instructions:
+
+1. Go to https://console.cloud.google.com/apis/credentials
+
```

Comment:
Maybe we could add the first step we had to do

Reply:
Done in c72f638e3